vaadin-testbench-unit/src/main/java/com/vaadin/flow/component/combobox/ComboBoxWrap.java

```diff
@@ -75,7 +75,7 @@ public void selectItem(String selection) {
         final List<Y> filtered = suggestionItems.stream().filter(
                 item -> selection.equals(itemLabelGenerator.apply(item)))
                 .collect(Collectors.toList());
-        if (filtered.size() > 1 || filtered.isEmpty()) {
+        if (filtered.size() != 1 || filtered.isEmpty()) {
```

Comment:
I think ` || filtered.isEmpty()` can be removed.

Reply:
done